lib/metadata.js

```diff
@@ -204,9 +209,17 @@ Metadata.prototype.generateOidcPEM = function generateOidcPEM(kid) {
     const exponent = new Buffer(key.e, 'base64');
 
     pubKey = aadutils.rsaPublicKeyPem(modulus, exponent);
+    foundKey = true;
+    
     return pubKey;
```

Comment:
1. should we just return 'true'. Why return pubKey?
2. do we need the 'const modulus = new Buffer(key.n, 'base64'); and const exponent = new Buffer(key.e, 'base64'); 
or can we just pass them as parameters as in:
pubKey = aadutils.rsaPublicKeyPem(new Buffer(key.n, 'base64'), new Buffer(key.e, 'base64'));